docs/en_US/Compression/v2_pruning_algo.rst

```diff
@@ -55,6 +55,29 @@ User configuration for Level Pruner
 
 .. autoclass:: nni.algorithms.compression.v2.pytorch.pruning.LevelPruner
 
+Performance Test
```

Comment:
all these performance numbers are obtained from our example code?

Reply:
Yes, the seed was set,  and some hyperparameters may change.

Reply:
is it better to also provide the hyper-parameters that you use?

Reply:
Now this has been provided.